Sources/PostgreSQL/Data/PostgreSQLData+String.swift

```diff
@@ -48,10 +48,10 @@ extension String: PostgreSQLDataConvertible {
                     
                     /// depending on our offset, append the string to before or after the decimal point
                     if offset < metadata.weight.bigEndian + 1 {
-                        integer += string
+                        integer += offset == 0 ? string : String(repeating: "0", count: 4 - string.count) + string
```

Comment:
Breaking this up into one or two lines with some comments would make it a lot more readable I think